zboxcore/sdk/chunked_upload_option.go

```diff
@@ -44,13 +44,13 @@ func WithThumbnailFile(fileName string) ChunkedUploadOption {
 }
 
 // WithChunkSize set custom chunk size. ignore if size <=0
-func WithChunkSize(size int64) ChunkedUploadOption {
-	return func(su *ChunkedUpload) {
-		if size > 0 {
-			su.chunkSize = size
-		}
-	}
-}
+// func WithChunkSize(size int64) ChunkedUploadOption {
+// 	return func(su *ChunkedUpload) {
+// 		if size > 0 {
+// 			su.chunkSize = size
+// 		}
+// 	}
+// }
```

Comment:
Delete commented code since they will not be used

Reply:
fixed